### DISCUSSION TITLE
Having trouble using custom font in QuestPDF

### DISCUSSION THREAD

**Discussion by dedmytas** [QuestPDF/QuestPDF#1056] [Q&A] [answered]
Hi,

Situation: trying to make a simple console application and trying to use custom fonts.

After registering the font and trying to use it, QuestPDF says that the typeface could not be found.

Images attached.

![image](https://github.com/user-attachments/assets/aab1a04c-4b0e-4b55-8a5f-90593f1fa681)

![image](https://github.com/user-attachments/assets/0f0b3d9c-9d88-459b-91f0-4d2f0ef96ed0)

If the font-family name is wrong, then how do I know the correct one?

I got the fonts from here: https://fontsgeek.com/fonts/Gotham-Narrow-Bold

Thank you.

**Comment by dedmytas** [comment, marked as answer]
Disregard my dumb post entirely, I am blind and cannot read.

Was using wrong method to register font.

Have to use `FontManager.RegisterFontWithCustomName("GothamNarrowBold", font1);`

**Comment by DawidRobertNowak** [comment]
Hello,

Glad you were able to find the solution! We are closing this thread as resolved. Thank you for sharing your answer — it may help others with the same question.

Thank you!
